src/main/resources/org/jfrog/hudson/ArtifactoryBuilder/global.jelly

```diff
@@ -123,16 +124,62 @@
 
                     <f:block>
                         <div align="right">
-                            <f:repeatableDeleteButton />
+                            <f:repeatableDeleteButton/>
                         </div>
                     </f:block>
 
                 </table>
             </f:repeatable>
         </f:entry>
+        <f:block>
+            <br/>
+        </f:block>
+        <f:entry title="Pipelines server" description="Send job results to JFrog Pipelines.">
+            <table width="100%">
+                <f:entry title="Callback URL"
+                         help="/plugin/artifactory/help/ArtifactoryBuilder/help-pipelinesCbkUrl.html">
+                    <f:textbox field="pipelinesCbkUrl" value="${descriptor.pipelinesServer.cbkUrl}"/>
+                </f:entry>
+
+                <f:advanced>
+                    <f:entry>
+                        <f:entry title="Connection Timeout" field="pipelinesTimeout"
```

Comment:
Connection Timeout --> Connection timeout

(should also fix this for the Artifactory servers configuration.)